bootstrap/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License
+
+Copyright (c) 2004-2010, Sun Microsystems, Inc., Kohsuke Kawaguchi,
```

Comment:
This (and the one below you commented) is because I copied this file from `core/pom.xml`. I don't know how much of pom is copyrightable, but I think I'll just leave it in.